Makefile

```diff
@@ -92,8 +91,8 @@ help:
 	@echo "- deploybranch       Deploys current branch to test (note: takes code from github)"
 	@echo "- deploybranchint    Deploys current branch to test and int (note: takes code from github)"
 	@echo "- deploybranchdemo   Deploys current branch to test and demo (note: takes code from github)"
-	@echo "- ol                 Update ol.js and ol-debug.js"
-	@echo "- node_modules_prod  Update productive depedencies in package.json"
+	@echo "- ol3cesium          Update ol3cesium.js, ol3cesium-debug.js, Cesium.min.js and Cesium folder"
+	@echo "- libs               Update js librairies used in index.html, see npm packages defined in section 'dependencies' of package.json"
```

Comment:
Since the description is up to date, i prefer to leave as it is. Short name is fatser to type:)